ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/ReportReviewForm.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { ReactElement } from 'react';
+
+import { ReportFormValues } from 'Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues';
+import {
+    Card,
+    CardBody,
+    CardTitle,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Divider,
+    Flex,
+    FlexItem,
+    PageSection,
+    Text,
+    TextContent,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import { getDate } from 'utils/dateUtils';
+import VulnerabilitySeverityIconText from '../components/VulnerabilitySeverityIconText';
+import { cveStatusLabelMap, cvesDiscoveredSinceLabelMap, imageTypeLabelMap } from '../utils';
+import exampleReportsCSVData from '../exampleReportsCSVData';
+
+export type ReportReviewFormParams = {
+    formValues: ReportFormValues;
+};
+
+function ReportReviewForm({ formValues }: ReportReviewFormParams): ReactElement {
+    const cveSeverities =
+        formValues.reportParameters.cveSeverities.length !== 0 ? (
+            formValues.reportParameters.cveSeverities.map((severity) => (
+                <li key={severity}>
+                    <VulnerabilitySeverityIconText severity={severity} />
+                </li>
+            ))
+        ) : (
+            <li>None</li>
+        );
+    const cveStatuses =
+        formValues.reportParameters.cveStatus.length !== 0 ? (
+            formValues.reportParameters.cveStatus.map((status) => (
+                <li key={status}>{cveStatusLabelMap[status]}</li>
+            ))
+        ) : (
+            <li>None</li>
+        );
+    const imageTypes =
+        formValues.reportParameters.imageType.length !== 0 ? (
+            formValues.reportParameters.imageType.map((type) => (
+                <li key={type}>{imageTypeLabelMap[type]}</li>
+            ))
+        ) : (
+            <li>None</li>
+        );
+
+    return (
+        <>
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Flex direction={{ default: 'column' }} className="pf-u-py-lg pf-u-px-lg">
+                    <FlexItem>
+                        <Title headingLevel="h2">Review and create</Title>
+                    </FlexItem>
+                </Flex>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection
+                variant="light"
+                padding={{ default: 'noPadding' }}
+                className="pf-u-py-lg pf-u-px-lg"
+            >
+                <Flex direction={{ default: 'column' }}>
+                    <FlexItem>
+                        <Title headingLevel="h3">Report parameters</Title>
+                    </FlexItem>
+                    <FlexItem flex={{ default: 'flexNone' }}>
+                        <DescriptionList
+                            columnModifier={{
+                                default: '2Col',
```

Comment:
Nit for all those users running ACS on their phone 😅: Could we do this here and in the section below?
```suggestion
                                default: '1Col',
                                sm: '2Col',
```

Reply:
Did you mean default is `2Col` and sm is `1Col`? There was a weird issue where it wouldn't go back to `2Col` when I stretched it out again. I added and md as well and seems to work now